core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -0,0 +1,40 @@
+PLATFORM_FLAVOR ?= npcm845x
+
+ifeq ($(PLATFORM_FLAVOR),npcm845x)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+CFG_ARM64_core ?= y
+endif #npcm845x
+
+CFG_USER_TA_TARGETS ?= ta_arm64
+
+ifeq ($(platform-debugger-arm),1)
+# ARM debugger needs this
+platform-cflags-debug-info = -gdwarf-2
+endif
+
+CFG_WITH_ARM_TRUSTED_FW ?= y
```

Comment:
Can this be used without TF-A? If not please force it to `y` and remove the unused code in `main.c`.